src/tests/counter/Counter.spec.js

```diff
@@ -144,6 +144,23 @@ it("can removeState", () => {
   });
 });
 
+it("properly reports action error", () => {
+  const counter = TestRenderer.create(<Counter />);
+  const button = counter.root.findByProps({
+    className: "actionWithoutReducer"
+  });
+
+  act(() => {
+    // ReactDOM will still output an error to the console; let's hide that for this test
+    const tmp = console.error;
+    console.error = function() {};
+    expect(() => button.props.onClick()).toThrow(
+      `action "actionWithoutReducer" must return a reducer function; instead got "object"`
+    );
+    console.error = tmp;
```

Comment:
I don't think you need to mess with `console.error` here - `toThrow` should gobble up the error.

Reply:
It does not.

Reply:
AH, so it's different from `index.spec.js`, for example, because it's the ReactDOM that's handling the error. Okay, fair enough.